example/main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
probably better to update the current `Example`: https://github.com/caarlos0/env/blob/06b4d47785c9245e4424541ecb188bd426bda4dd/env_test.go#L1306

Reply:
maybe would be good to put all the `Example.*` functions in a `examples_test.go` so its easier to find though

Reply:
😆  done

Reply:
awesome, thank you!